yaksh/views.py

```diff
@@ -663,9 +681,11 @@ def courses(request):
     ci = RequestContext(request)
     if not is_moderator(user):
         raise Http404('You are not allowed to view this page')
-    courses = Course.objects.filter(creator=user, is_trial=False)
+    demo_user = User.objects.get(username="demo_user")
```

Comment:
You will need to consider the possibility that the management command has not been executed. Please add a try/catch or an if/else to handle the case where the demo_user does not exist